.github/workflows/ci.yml

```diff
@@ -9,19 +9,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10.x, 12.x]
+        node: [12.x]
```

Comment:
One build seems to take only around 20s
So we can use a `matrix.node` of 10.x (min LTS), 12.x (current LTS) and 14.x (newest) and also maybe 13.x if we want to

Reply:
Ok. I will add these 4 then.

Reply:
Further information: https://nodejs.org/en/about/releases/

Reply:
13 may not be too relevant.